test/support/test_config.h

```diff
@@ -117,6 +117,8 @@
 #endif
 #endif //!defined(_ENABLE_RANGES_TESTING)
 
+#define _ENABLE_STD_RANGES_TESTING (_ONEDPL___cplusplus >= 202002L)

```

Comment:
`_ONEDPL___cplusplus` is an implementation detail. Please use `__cplusplus` and `_MSVC_LANG` here. See also https://github.com/oneapi-src/oneDPL/pull/1568#discussion_r1593947643

Reply:
Probbaly, does it make sense to use the following definition in `onedpl_config.h `??
`#define _ONEDPL_CPP20_RANGES_PRESENT (__cplusplus >= 202002L || _MSVC_LANG >= 202002L) && __cpp_lib_ranges >= 201911L `
?
(https://en.cppreference.com/w/cpp/feature_test)